src/structures/Message.js

```diff
@@ -386,6 +379,26 @@ class Message extends Base {
     return this.channel.guild ?? null;
   }
 
+  /**
+   * Whether this message has a thread associated with it
+   * @type {boolean}
+   * @readonly
+   */
+  get hasThread() {
+    return this.flags.has(MessageFlags.FLAGS.HAS_THREAD);
+  }
+
+  /**
+   * The thread started by this message
+   * <info>This can be null when the message has a thread if the cache no longer has the thread.
```

Comment:
Isn’t this already implicit for all getters of this nature? I don’t see a reason to add this info. Otherwise LGTM

Reply:
I added it because threads are for more likely to be out of cache (once they're archived at least). Although by the point they're archived for 4 hours (assuming default options here) the message has probably dropped out of cache too.

Reply:
Yeah that sounds better